sable_ircd/src/capability/mod.rs

```diff
@@ -66,6 +67,7 @@ define_capabilities! (
     ClientCapability
     {
         // Stable caps
+        MessageTags:            0x01 => ("message-tags", true),
```

Comment:
This cap got removed earlier (#54) because the spec includes TAGMSG and client-only tags if this is negotiated. Do we want to put it back without those things?

Reply:
oh you're right, I should advertise `CLIENTTAGDENY=*`